src/lib/Syntax.hs

```diff
@@ -728,6 +727,9 @@ instance Ord SourceBlock where
 
 -- === Synonyms ===
 
+varType :: Var -> Type
+varType = varAnn
```

Comment:
Any reason to have `varType` instead of just using `varAnn`? Maybe the more specific type conveys intent?

Reply:
Yeah, that was my idea. `varAnn` does not indicate that it returns a type (and in some cases it doesn't!), but this is how we _usually_ use it in Dex